CIAC-14385 (#40882)
demisto/content: Packs/Azure/ReleaseNotes/1_0_7.md

#### Integrations

##### Azure

> - Added an error handling mechanism (the change has no impact on this integration).

Reviewer: ```suggestion
- Added an error handling mechanism for automations (the change has no impact on this integration).
```

> - Added an error handling mechanism for automations (the change has no impact on this integration).

Reviewer 2: ```suggestion
- Added an error handling mechanism (the new mechanism doesn't impact the integration's behavior).
```
